bindings/python3/econf.py

```diff
@@ -0,0 +1,996 @@
+"""
+Econf provides functionality for interacting with Key-Value config files, like getting and setting values for read config files.
+
+For more information please have a look at the API
+"""
+import ctypes.util
+from enum import Enum
+from dataclasses import dataclass
+from typing import *
+from ctypes import *
+
+LIBNAME = ctypes.util.find_library("econf")
+LIBECONF = CDLL(LIBNAME)
+
+
+@dataclass
+class EconfFile:
+    """
+    Class which points to the Key Value storage object
+    """
+
+    _ptr: c_void_p
+
+    def __del__(self):
+        free_file(self)
+
+
+class Econf_err(Enum):
+    ECONF_SUCCESS = 0
```

Comment:
If you remove the ECONF_ prefix, you can use "EconfError.NOKEY", for example. The class is providing a namespace.